certgen/certProcessor/src/main/java/org/incredible/certProcessor/CertificateFactory.java

```diff
@@ -60,11 +61,18 @@ public CertificateExtension createCertificate(CertModel certModel, Map<String, S
          * badge class object
          * **/
 
-        badgeClassBuilder.setName(certModel.getCourseName()).setDescription(certModel.getCertificateDescription())
+        badgeClassBuilder.setName(certModel.getCertificateName()).setDescription(certModel.getCertificateDescription())
                 .setId(properties.get(JsonKey.BADGE_URL)).setCriteria(certModel.getCriteria())
                 .setImage(certModel.getCertificateLogo()).
                 setIssuer(issuerBuilder.build());
 
+        /**
+         * Training evidence
+         */
+        if (StringUtils.isNotBlank(certModel.getCourseName())) {
+            trainingEvidenceBuilder.setId(uuid).setName(certModel.getCourseName());
```

Comment:
An evidence has a subject - which is the recipient that has got this. Can you clarify if the uuid here is recipient id or cert id ?

Reply:
this uuid is base url+certId  (http://localhost:9000/certs/0125450863553740809/89047030-c28b-4f05-8168-2eae31cafae)